static/app/views/settings/project/server-side-sampling/index.tsx

```diff
@@ -16,18 +13,11 @@ export default function ServerSideSamplingContainer({project}: Props) {
 
   return (
     <Feature
-      features={['server-side-sampling', 'server-side-sampling-ui']}
+      requireAll={false}
+      features={['dynamic-sampling-basic', 'server-side-sampling-ui']} // <-- the "server-side-sampling-ui" feature will be removed in the future
```

Comment:
```suggestion
      features={[
            'organization:server-side-sampling',
            project.dynamicSampling?.rules.length
              ? 'organization:server-side-sampling-ui'
              : 'organization:dynamic-sampling-basic',
          ]}
```

Once we go GA `organization:server-side-sampling-ui` will be true for everyone.
We want to display this page for people on:
1. paid plans
2. free plans who already have a rule because they were part of LA program (we will communicate them via email and UI that they have a month or two to either upgrade or will loose the feature)

Reply:
I've added this logic to a different place https://github.com/getsentry/getsentry/blob/3ff45fd40f8391cdf03827e7dc41b034f41ae10b/static/getsentry/gsApp/components/features/disabledDynamicSampling.tsx#L87

Reply:
We still need to check the existence of `organization:server-side-sampling` AND either one flag or the other based on existence of the rules.

Reply:
that's okay - if they don't have any rules created we can start showing plan upsells